NuGet.Docs/Release-Notes/NuGet-3.2.1.md

```diff
@@ -0,0 +1,26 @@
+
+# NuGet 3.2.1 Release Notes
+
+[NuGet 3.2 Release Notes](nuget-3.2)
+
+NuGet 3.2.1 for the command-line was released October 7, 2015 with a handful of optimizations and fixes for the 3.2 release and is available from both [dist.nuget.org](http://dist.nuget.org/index.html) and the [Visual Studio Gallery](https://visualstudiogallery.msdn.microsoft.com/5d345edc-2e2d-4a9c-b73b-d53956dc458d?SRC=Home). 
+
+## Improvements
+
+* NuGet now uses the configuration file with the original casing of `NuGet.Config`.  This is important on case-sensitive operating systems [1427](https://github.com/NuGet/Home/issues/1427)
+* NuGet restore will now ignore dnx projects (*.xproj) that should be processed with `dnu` [1227](https://github.com/NuGet/Home/issues/1227)
+* Optimized network utilization when working with index.json and package registration data [1426](https://github.com/NuGet/Home/issues/1426)
+* Improved resource download handling to be more robust with v2 services [1448](https://github.com/NuGet/Home/issues/1448)
+
+## Fixes
+
+* NuGet update correctly updates csproj/vcxproj references [1483](https://github.com/NuGet/Home/issues/1483)
+* Now preventing a local .nuget folder from being created when a userprofile folder cannot be located [1531](https://github.com/NuGet/Home/issues/1531)
```

Comment:
That's not accurate - We detect userprofile from environment variables, when it cannot be detected from SpecialFolders.UserProfile construct.